src/index.js

```diff
@@ -1,4 +1,47 @@
-import _ from "lodash";
-import "./style.css";
+const todoTask = [
+  {
+    index: 1,
+    description: "Complete one coding challenge",
+    completed: false,
+  },
+  {
+    index: 2,
+    description: "Daily task of agency app",
+    completed: false,
+  },
+];
 
-console.log("Hello, from Webpack!");
+class ToDo {
+  constructor(todo, wrapperElem) {
+    this.todo_list = todo;
+    this.wrapper_elem = wrapperElem;
+  }
+
+  completeTodoContainer = "";
+
+  insertTodo(singleTodo) {
+    this.complete_todo_container += `
+      <article class="single-todo">
+        <input type="checkbox" name=${singleTodo.index} class="checkbox-todo"/>
+        <p class="description-todo">${singleTodo.description}</p>
+        <div class="organize-todo">...</div>
+      </article>
+    `;
+  }
+
+  showTodo() {
+    this.todo_list.forEach((element) => {
+      this.insertTodo(element);
+
+      const targetElem = document.querySelector(this.wrapper_elem);
+      targetElem.innerHTML = this.complete_todo_container;
+    });
+  }
+
+  run() {
```

Comment:
- [ ] Kindly style your app according to the design provided.

  ![image](https://github.com/Aqurds/t-d-l/assets/81468675/89b28b45-05f7-4b76-b980-da831bbea69d)

  ![kdkd](https://github.com/Aqurds/t-d-l/assets/81468675/d47ae763-8b80-49c2-9f35-7da38caf3f99)